app/controllers/idv/agreement_controller.rb

```diff
@@ -32,7 +32,11 @@ def update
       if result.success?
         idv_session.idv_consent_given = true
 
-        redirect_to idv_hybrid_handoff_url
+        if IdentityConfig.store.in_person_proofing_opt_in_option
+          redirect_to idv_how_to_verify_url
```

Comment:
It makes a lot of sense to branch off this new page from the Agreement page! Lmk if you want to pair on any of this - setting up an A/B test, wrangling changes to the flow, etc.